### ISSUE TITLE
pod trunk push: xcodebuild: error: 'App.xcworkspace' does not exist.

### ISSUE THREAD

**Issue by hkellaway** [CocoaPods/CocoaPods#5908]
# Report

## What did you do?

`pod trunk push`

using 1.1.0.rc.2

## What did you expect to happen?

New version should be pushed.

## What happened instead?

The `push` fails with the error `xcodebuild: error: 'App.xcworkspace' does not exist.`

### Stack

```
   CocoaPods : 1.0.1
        Ruby : ruby 2.1.1p76 (2014-02-24 revision 45161) [x86_64-darwin12.0]
    RubyGems : 2.2.2
        Host : Mac OS X 10.11.6 (15G31)
       Xcode : 8.0 (8A218a)
         Git : git version 2.8.4 (Apple Git-73)
Ruby lib dir : /Users/xxx/.rvm/rubies/ruby-2.1.1/lib
Repositories : master - https://github.com/CocoaPods/Specs.git @ 00ff3e0bb44dd52598376b3311f4256c87dd3840
```

### Installation Source

```
Executable Path: /Users/xxx/.rvm/gems/ruby-2.1.1/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.1
cocoapods-keys        : 1.6.0
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.0.0
cocoapods-try         : 1.1.0
```

## Project that demonstrates the issue

https://github.com/hkellaway/Gloss/tree/master

**Comment by hkellaway** [comment]
i did read through this similar issue (https://github.com/CocoaPods/CocoaPods/issues/5843) - but the suggestions did not help

**Comment by magic3584** [comment]
The same with me #5900

**Comment by neonichu** [comment]
You need to use the latest release candidate instead of 1.0.1 if you are using Xcode 8.

**Comment by hkellaway** [comment]
ah! I thought I had 1.1.0 not 1.0.1 😅 thanks

**Comment by Abhishek9634** [comment]
same issue with 1.0.1

**Comment by benasher44** [comment]
This is fixed in 1.1.0. Please use `gem install cocoapods --pre` to get the latest release candidate.

**Comment by Abhishek9634** [comment]
YES rc3 works for me
